Content/Guides/manpages/loggen.1.html

```diff
@@ -4,362 +4,176 @@
+				</li>
+				<li>
+					<p><span class="Code">dont-parse</span> or <span class="Code">-d</span></p>
+					<p>Do not parse the lines read from the input files, send them as received.</p>
+				</li>
+				<li>
+					<p><span class="Code">--help</span> or <span class="Code">-h</span></p>
+					<p>Display a brief help message.</p>
+				</li>
+				<li>
+					<p><span class="Code">--idle-connection	&lt;number-of-connections&gt;</span>
+					</p>
+					<p>Number of idle connections <span class="Code">loggen</span> will establish to the destination. Note that <span class="Code">loggen</span> will not send any messages on idle connections, but the connection is kept open using keep-alive messages. This option is usable only when using TCP or TLS connections to the destination. See also the <span class="Code">--active-connections</span> option. Default value: 0</p>
+				</li>
+				<li>
+					<p><span class="Code">--inet</span> or <span class="Code">-i</span></p>
+					<p>Use the TCP (by default) or UDP (when used
+						together with the <span class="Code">--dgram</span> option)
+					protocol to send the messages to the target.</p>
+				</li>
+				<li>
+					<p><span class="Code">--interval &lt;seconds&gt;</span> or <span class="Code">-I &lt;seconds&gt;</span></p>
+					<p>The number of seconds <span class="Code">loggen</span> will run. Default value: 10</p>
+					<div class="Note">
+						<p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> Note that when the <span class="Code">--interval</span> and <span class="Code">--number</span> are used together, <span class="Code">loggen</span> will send messages until the period set in <span class="Code">--interval</span> expires or the amount of messages set in <span class="Code">--number</span> is reached, whichever happens first.</p>
+					</div>
+				</li>
+				<li><p><span class="Code">--ipv6</span> or <span class="Code">-6</span></p>
+					<p>Specify the destination using its IPv6 address. Note that the destination must have a real IPv6 address.</p>
+				</li>
+				<li><p><span class="Code">--loop-reading</span> or <span class="Code">-l</span></p>
+					<p>Read the file specified in <span class="Code">--read-file</span> option in loop: loggen will start reading from the beginning of the file when it reaches the end of the file.</p>
+				</li>
+				<li><p><span class="Code">--number &lt;number-of-messages&gt;</span> or <span class="Code">-n &lt;number-of-messages&gt;</span></p>
+					<p>Number of messages to generate.</p>
+					<div class="Note">
+						<p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> Note that when the <span class="Code">--interval</span> and <span class="Code">--number</span> are used together, <span class="Code">loggen</span> will send messages until the period set in <span class="Code">--interval</span> expires or the amount of messages set in <span class="Code">--number</span> is reached, whichever happens first.</p>
```

Comment:
```suggestion
						<p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> When <span class="Code">--interval</span> and <span class="Code">--number</span> are used together, <span class="Code">loggen</span> will send messages until the period set in <span class="Code">--interval</span> expires or the amount of messages set in <span class="Code">--number</span> is reached, whichever happens first.</p>
```

Reply:
Might be useful to put these identical notes into snippets. Also other "stock" paragraphs, like in the very end, e.g "For more information on syslog-ng PE/OSE, see XY documentation page...." could be used as snippets. If not now, then maybe in a next iteration. In a separate Jira task maybe... I've seen identical paragraphs in the other flare topics too.